lib/checkPullRequestReview.js

```diff
@@ -1,3 +1,4 @@
+const { context } = require('@actions/github');
```

Comment:
Is this required? If yes, why wasn't it present before?

Reply:
It is not being used. Removed.